apps/solver/base.hu.i18n

```diff
@@ -1,29 +1,29 @@
 SolverApp = "Egyenletek"
-SolverAppCapital = "Egyenletek"
-AddEquation = "Egyenlet hozzáadása"
-ResolveEquation = "Oldja meg az egyenletet"
-ResolveSystem = "A rendszer megoldása"
+SolverAppCapital = "EGYENLETEK"
+AddEquation = "Egyenlet hozzáadás"
+ResolveEquation = "Egyenletet megoldása"
+ResolveSystem = "Rendszer megoldása"
 UseEquationModel = "Használjon egyenleti sablont"
-RequireEquation = "A bemenetnek egyenletnek kell lennie"
-UnrealEquation = "Unreal egyenlet"
-UndefinedEquation = "Definiálatlan egyenlet"
-TooManyVariables = "Túl sok ismeretlen"
+RequireEquation = "A bemenetnek csak egyenlet lehet"

```

Comment:
We can write "A bemenet csak egyenlet lehet" or " A bemenetnek egyenletnek kell lennie" both are correct.